### DISCUSSION TITLE
Lane connector towards a roundabout?

### DISCUSSION THREAD

**Discussion by totte1015** [krzychu124/Traffic#10] [Q&A]
Why does it say "modifying lane connections at a selected intersection is not supported" in a roundabout??
Its very silly since its very much needed, if I have two lane roundabout I want to be able to edit where cars go :(

**Comment by krzychu124** [comment]
> Why does it say "modifying lane connections at a selected intersection is not supported" in a roundabout?? Its very silly since its very much needed, if I have two lane roundabout I want to be able to edit where cars go :(

If you have dev ui enabled, check _Gizmos_ tab -> enable _Lane Debug System_ then enable at least 3 first options (plus the last one for priorities for better visibility) and see how many lane connections it generates (all would have to be handled by lane connector, and all are generated on the fly when you plop roundabout upgrade or touch any road connected to it afterwards). 
For just 3 legs (2W2L) roundabout it's +30 lane connections, each additional 2W2L leg adds another 10 lane connections if not more, since there are mid-points and in certain conditions mid-points between mid-points.
You still can create your own roundabout and make connections as you wish, especially with _Road Builder_ mod
